tests/test_connector.py

```diff
@@ -1,100 +1,145 @@
+import json
 from logging import getLogger
 from unittest import mock
 
 from py42.exceptions import Py42UnauthorizedError
+from py42.response import Py42Response
 from pytest import fixture
+from requests import Response
 
-import code42_connector
+from code42_connector import Code42Connector
 from phantom.action_result import ActionResult
 import phantom.app as phantom
 import py42.sdk
+from py42.services.users import UserService
 
 logger = getLogger(name=__name__)
 
 NULL_VALUE = object()
 
 
-@fixture
+@fixture(autouse=True)
 def mock_py42_client(mocker):
     client = mocker.MagicMock(spec=py42.sdk.SDKClient)
-    client.users = mocker.MagicMock(spec=py42.services.users.UserService)
+    client.users = mocker.MagicMock(spec=UserService)
+    mocker.patch("py42.sdk.from_local_account", return_value=client)
     return client
 
 
+@fixture
+def mock_result_adder(mocker):
+    return mocker.patch("phantom.base_connector.BaseConnector.add_action_result")
+
+
 @fixture
 def connector():
-    connector = code42_connector.Code42Connector()
+    connector = Code42Connector()
     return connector
 
 
 def create_fake_connector(action_identifier):
     def fake_get_action_identifier():
         return action_identifier
 
-    connector = code42_connector.Code42Connector()
+    connector = Code42Connector()
     connector.get_action_identifier = fake_get_action_identifier
     return connector
 
 
 @fixture
 def test_connectivity_connector(mock_py42_client):
-    connector = create_fake_connector("test_connectivity")
+    connector = create_fake_connector(Code42Connector.TEST_CONNECTIVITY_ACTION_ID)
     connector._client = mock_py42_client
     return connector
 
 
 class TestCode42Connector(object):
```

Comment:
I think it probably makes sense to split the tests up by action category, otherwise this will become a monster

Reply:
Should they be in different files or just different classes?
If different files, we can move the fixtures to a conftest and that will even more-so lighten up their monstrosities

Reply:
yeah that's exactly what I was thinking, different files